### ISSUE TITLE
Switch markdown rendering to Markdig.Wpf

### ISSUE THREAD

**Issue by RichiCoder1** [chocolatey/ChocolateyGUI#492]
Currently we use CefSharp to render markdown as there wasn't any good alternatives. However, the excellent https://github.com/Kryptos-FR/markdig.wpf project has popped up which appears to do exactly what we need with high fidelity. It means we won't perfectly match the fidelity of the web package viewer, but it will greatly decrease the size and moderately decrease the load time of Gui.

**Comment by Kryptos-FR** [comment]
Hi.

Markdig.Wpf author here. I'm glad to see that you like my library.

I actually had no idea that it was used in a real project somewhere. I wrote it initially as a pet project when @xoofx (markdig's author) was a coworker at my previous company.

I haven't had the time recently to work on it (although I did accept one or two pull requests), but if you think something is missing, don't hesitate to open an issue or contribute with a pull-request.

Cheers,
Nicolas
